nimble/include/nimble/hci_common.h

```diff
@@ -1153,8 +1153,8 @@ struct ble_hci_le_subrate_req_cp {
 #define BLE_HCI_OCF_LE_SET_EXT_ADV_PARAM_V2             (0x007F)
 struct ble_hci_le_set_ext_adv_params_v2_cp {
     struct ble_hci_le_set_ext_adv_params_cp cmd;
-    uint8_t pri_phy_opt;
-    uint8_t sec_phy_opt;
+    uint8_t primary_phy_opt;
```

Comment:
this should be left as is,  on HCI we have pri_phy and sec_phy (and this is internal anyway)

Reply:
Done